lib/commands/web.js

```diff
@@ -41,17 +44,26 @@ extensions.getSafariIsIphone = _.memoize(async function getSafariIsIphone () {
   return true;
 });
 
-extensions.getSafariIsIphoneX = _.memoize(async function getSafariIsIphone () {
-  try {
-    const script = 'return {height: window.screen.availHeight, width: window.screen.availWidth};';
-    const {height, width} = await this.execute(script);
-    // check for the correct height and width
-    const [portraitHeight, portraitWidth] = height > width ? [height, width] : [width, height];
-    return (portraitHeight === IPHONE_X_HEIGHT && portraitWidth === IPHONE_X_WIDTH) ||
-           (portraitHeight === IPHONE_XR_HEIGHT && portraitWidth === IPHONE_XR_WIDTH);
+extensions.getSafariDeviceSize = _.memoize(async function getSafariDeviceSize () {
```

Comment:
what if screen orientation gets changed during the test?

Reply:
This is just for deciding on the device, and it is normalized to portrait. So it does not matter.